client/src/ui/modules/military/battle-view/BattleProgressBar.tsx

```
@@ -136,14 +136,14 @@ export const BattleProgressBar = ({
     <motion.div
       initial="hidden"
       animate="visible"
-      className=""
+      className="w-2/3 mx-auto"
       variants={{
         hidden: { y: "100%" },
         visible: { y: "0%", transition: { duration: 0.5 } },
       }}
     >
       {!isNaN(Number(attackingHealthPercentage)) && !isNaN(Number(defendingHealthPercentage)) && (
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)